### DISCUSSION TITLE
Can a second PivotTable be added to an Excel doc from data in an existing worksheet?

### DISCUSSION THREAD

**Discussion by BryHall** [dfinke/ImportExcel#1568] [Q&A] [answered]
I'm able to export my data and generate a PivotTable in a new sheet just fine with a single PowerShell command. 

`$ExtendedAssignments | Export-Excel @ExcelFileParams -KillExcel -WorksheetName 'Azure RBAC Assignments' -AutoNameRange -PivotRows 'Notes','RoleDefinitionName','SignInName','ObjectType','Scope' -PivotData @{'ObjectId' = 'count'} -PivotTableName 'Findings Pivot' -ClearSheet`

However, I'd like to create a second PivotTable in a new sheet from the same source data in 'Azure RBAC Assignments'.  Is there a way to do this, or must I export the same data again to a different sheet and generate that PivotTable at that time?

**Comment by dfinke** [comment, marked as answer]
Have a look here: https://github.com/dfinke/ImportExcel/tree/master/Examples/PivotTable

Let me know this *does not* fit your scenario.

**Reply by BryHall** [reply]
Thank you, this looks like what I needed!
